Extensions/DateTimeExtensions.cs

```diff
@@ -10,11 +10,13 @@ public static class DateTimeExtensions
 
         public static string ToIsoDateString(this DateTime dateTime) => dateTime.ToString("yyyy-MM-dd");
 
-        public static string ToIsoDateString(this DateTime? dateTime) => dateTime?.ToIsoDateString() ?? string.Empty;

```

Comment:
You shouldn't remove any extension method. You may not use that in your project, but this submodule is not COLI-exclusive.